### ISSUE TITLE
Show the just uploaded data when uploading timeseries .csv or rasters .tiff

### ISSUE THREAD

**Issue by Derryrover** [nens/lizard-management-client#847]
Maybe in the popup snackbar that says "succes" also a link to the just uploaded data?
Problem (snackbar dissappears maybe too fast)

Link might be in portal or api.
How do we get this link?
Can we generate this clientside based on the data or do we need a backend response?

**Comment by hoanphungt** [comment]
The raster upload generates an async task and I believe that currently there is no way to view the uploaded data of the raster using the API yet. Is it correct? @lexvand 

For timeseries csv upload, the uploaded events of the timeseries can be viewed here: `/api/v4/timeseries/{uuid}/events/`. We can certainly include this link in the snackbar and make it stay longer.

**Comment by lexvand** [comment]
@hoanphungt That's correct. It is very difficult to make a good solution for either rasters (indeed async task and what data to show?) or timeseries (what time range to show? how to show it?). 

For now we're not going to implement something like this.
